text/0000-compiler-team-trusted-contrib-maintainer.md

```diff
@@ -0,0 +1,381 @@
+trusted contributor will be moved into alumni status. At any point in the
+future, they can ask to be re-instated at the trusted contributor level if they
+desire.
+
+[Compiler Development Guide]: https://rustc-dev-guide.rust-lang.org/
+
+Team Members
+------------
+[team-members]: #team-members
+
+Trusted contributors are eligible to become team members after they have
+continued to contribute actively for a year. Trusted contributors can contact
+team leads or will be contacted by team leads to enquire about promotion to team
+membership. Trusted contributors who are eligible for team membership do not
+have to become team members.
+
+Unlike trusted contributors, team members are expected to consider themselves
+as *maintainers* of the compiler - put otherwise, to be invested in the quality
+of the compiler codebase and overall health of the compiler team, independent
+of their own projects. Team membership is primarily intended to recognise and
+encourage participation in activities which are vital to the success of the
+compiler team and broader project.
+
+Team members are expected to participate in the ongoing maintainance tasks
+that the compiler team is responsible for (with all of the expected caveats
+for vacation time, mental health breaks, etc) - listed as [responsibilities]
+[responsibilities] below. However, not all contributors need to participate
+in these responsibilities to an equal degree. Contributors should participate
+in these tasks to the degree that they are able - volunteers are not expected
+to participate as much as those employed to work on the compiler, for example.
+It is the responsibility of the compiler team leads to ensure that the ongoing
+maintenance tasks of the team can be completed sustainably.
+
+Team members have all of the same permissions and access as trusted
+contributors. Like trusted contributors, team members are considered members of
+the Rust project as a whole and are expected to follow the spirit of the [Code
+of Conduct][coc].
+
+Like trusted contributors, after inactivity for longer than a year, a
+contributor will be moved to alumni status. Members who are no longer able to
+help maintain the compiler but otherwise wish to continue contributing to the
+compiler can also be moved to alumni status and retain their trusted contributor
+status. Alumni can ask to be reinstated in future.
+
+[coc]: https://www.rust-lang.org/policies/code-of-conduct
+
+Responsibilities
+----------------
+[responsibilities]: #responsibilities
+
+There are various responsibilities that a team member could take on to help the
+team. All team members must participate in at least one activity.
+
+Team members can get involved in any of these by contacting the team leads, by
+asking team members currently involved in these responsibilities, or by asking
+in any venue where these responsibilities are conducted (e.g. a Zulip stream).
+
+- Final comment period (FCP) reviewer
```

Comment:
I think it would be good to explicitly state some kind of "its expected that fcps are responded to by reviewers within Some Concrete Timeframe" regardless of whether that response is ticking the box or registering a concern or just "im still taking time reviewing this in depth". it tends to Feel Bad when you go onto an FCP and its been open for a while with seemingly no response from someone regardless of whether there's actually a good reason behind the scenes.

Reply:
I've added wording to this effect.